code/src/App.js

```diff
@@ -1,12 +1,36 @@
 import React from 'react'
 import data from './data.json'
 
+// IMPORT COMPONENTS
+import { Album } from './components/Album'
+import { Header} from './components/Header'
+
 console.log(data)
 
+// ALBUMS ARRAY 
+const albumsArray = data.albums.items
+
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <>
+    <header>
+      <Header />
+    </header>
+    <main className="albums-container">
```

Comment:
Maybe a good idea would have been to extract this albums container with its map function etc. to another component, since App.js is meant to be quite short and to contain only the components to render.